README.md

```diff
@@ -1,5 +1,4 @@
-# PDF.js [![Build Status](https://travis-ci.org/mozilla/pdf.js.svg?branch=master)](https://travis-ci.org/mozilla/pdf.js)
-
+# [PDF.js](https://mozilla.github.io/pdf.js/) &middot; [![Build Status](https://travis-ci.org/mozilla/pdf.js.svg?branch=master)](https://travis-ci.org/mozilla/pdf.js) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/mozilla/pdf.js/blob/master/LICENSE) [![npm version](https://img.shields.io/npm/v/pdfjs-dist)](https://www.npmjs.com/package/pdfjs-dist) [![npm-downloads](https://img.shields.io/npm/dm/pdfjs-dist)](https://npmcharts.com/compare/pdfjs-dist?minimal=true) [![Follow Twitter](https://img.shields.io/twitter/follow/pdfjs?style=social)](https://twitter.com/pdfjs)
```

Comment:
It's not really clear that we need to list all of these things at the top of the README, note how e.g. the license information is already listed at the *top* of https://github.com/mozilla/pdf.js

Furthermore, given how infrequent the PDF.js Twitter-account is updated (the last post was 4.5 years ago), it probably doesn't make sense to list that at the top of the README either!?

Hence I'd suggest simplifying this to only:
```suggestion
# [PDF.js](https://mozilla.github.io/pdf.js/) [![Build Status](https://travis-ci.org/mozilla/pdf.js.svg?branch=master)](https://travis-ci.org/mozilla/pdf.js) [![npm version](https://img.shields.io/npm/v/pdfjs-dist)](https://www.npmjs.com/package/pdfjs-dist)
```